.travis.yml

```diff
@@ -19,14 +15,18 @@ node_js:
     - '5.0'
 
 env:
-    - INFLUX_VERSION=0.9
-    - INFLUX_VERSION=0.10
+    - INFLUX_VERSION=0.9 JUTTLE_PACKAGE=juttle/juttle
+    - INFLUX_VERSION=0.9 JUTTLE_PACKAGE=juttle@0.4.x
+    - INFLUX_VERSION=0.10 JUTTLE_PACKAGE=juttle/juttle
+    - INFLUX_VERSION=0.10 JUTTLE_PACKAGE=juttle@0.4.x
```

Comment:
I don't see it being valuable to test the full cross product of node / influx / juttle versions. This will extend build times without really adding much in the way of coverage.

It seems sufficient to me to test half the builds with:

- node 4.2 influx 0.10 juttle release
- node 5.0 influx 0.10 juttle release
- node 4.2 influx 0.9 juttle release
- node 4.2 influx 0.10 juttle master